tasks/jest/matchers/toHaveNoViolations.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const axe = require('axe-core');
+
+const defaultOptions = {
+  rules: {
+    'document-title': {
+      enabled: false,
+    },
+    'html-has-lang': {
+      enabled: false,
+    },
+    'landmark-one-main': {
+      enabled: false,
+    },
+    'page-has-heading-one': {
+      enabled: false,
+    },
+    region: {
+      enabled: false,
+    },
+  },
+};
+
+function toHaveNoViolations(node, options = {}) {
+  return new Promise(resolve => {
+    axe.run(
+      node,
+      {
+        ...defaultOptions,
+        ...options,
+      },
+      (error, result) => {
+        if (error) {
+          throw error;
+        }
+
+        if (result.violations.length > 0) {
+          resolve({
+            message: () => format(result.violations),
+            pass: false,
+          });
+          return;
+        }
+
+        resolve({
+          pass: true,
+        });
+      }
+    );
+  });
+}
+
+function format(violations) {
```

Comment:
Would you be down to call this like `formatOutput`?

Reply:
For sure, feel free to use a commit suggestion block in the comment and I'll apply it for sure 👍